test/visual/z_svg_export.js

```diff
@@ -369,7 +369,7 @@
"L",442.481,783.612],["C",442.173,783.9549999999999,441.777,784.252,441.751,784.735],["z"]]},{"type":"path","version":"5.1.0","left":55.503,"top":85.9516,"width":2.9869,"height":8.776,"fill":{"type":"linear","coords":{"x1":481.066,"y1":785.465,"x2":480.953,"y2":793.102},"colorStops":[{"offset":1,"color":"rgb(160,137,44)","opacity":1},{"offset":0.9,"color":"rgb(85,68,0)","opacity":1},{"offset":0.78,"color":"rgb(80,68,22)","opacity":1},{"offset":0.607,"color":"rgb(160,137,44)","opacity":1},{"offset":0.467,"color":"rgb(255,255,255)","opacity":1},{"offset":0.299,"color":"rgb(200,171,55)","opacity":1},{"offset":0.24,"color":"rgb(160,137,44)","opacity":1},{"offset":0.096,"color":"rgb(120,103,33)","opacity":1},{"offset":0,"color":"rgb(211,188,95)","opacity":1}],"offsetX":-228.298,"offsetY":-835.244,"gradientUnits":"pixels","gradientTransform":[0.9334,0,0,0.8563,-219.064,163.965]},"stroke":"","scaleX":9.32,"scaleY":10.58,"angle":90,"flipY":true,"path":[["M",228.298,844.02],["L",230.868,842.937],["C",231.468,840.889,231.345,838.652,230.972,836.35],["L",228.352,835.244]]},{"type":"circle","version":"5.1.0","left":-168.6958,"top":-206.8033,"width":4.642,"height":4.642,"fill":{"type":"radial","coords":{"x1":193.676,"y1":141.252,"x2":193.676,"y2":141.252,"r1":0,"r2":4.082},"colorStops":[{"offset":1,"color":"rgb(0,0,0)","opacity":1},{"offset":0.9689999999999999,"color":"rgb(0,0,0)","opacity":1},{"offset":0.904,"color":"rgb(236,236,236)","opacity":1},{"offset":0.8740000000000001,"color":"rgb(77,77,77)","opacity":1},{"offset":0.8370000000000001,"color":"rgb(237,237,237)","opacity":1},{"offset":0.8169999999999998,"color":"rgb(0,0,0)","opacity":1},{"offset":0,"color":"rgb(0,0,0)","opacity":1}],"offsetX":2.321,"offsetY":2.321,"gradientUnits":"pixels","gradientTransform":[0.3487,0.4048,-0.4034,0.3475,-10.56,-127.518]},"stroke":"","scaleX":59.8,"scaleY":59.8,"radius":2.321},{"type":"circle","version":"5.1.0","left":-107.9658,"top":-147.8833,"width":3.578,"height":3.578,"fill":{"type":"linear","coords":{"x1":195.171,"y1":143.461,"x2":191.574,"y2":138.568},"colorStops":[{"offset":1,"color":"rgb(204,204,204)","opacity":1},{"offset":0.687,"color":"rgb(255,255,255)","opacity":1},{"offset":0,"color":"rgb(255,255,255)","opacity":1}],"offsetX":1.789,"offsetY":1.789,"gradientUnits":"pixels","gradientTransform":[0.5287,0,0,0.5287,-102.306,-74.736]},"stroke":"rgb(179,179,179)","strokeWidth":0.02,"strokeLineCap":"round","scaleX":59.8,"scaleY":59.8,"radius":1.789},{"type":"circle","version":"5.1.0","left":-14.2958,"top":-54.2033,"width":0.46,"height":0.46,"fill":{"type":"linear","coords":{"x1":656.429,"y1":320.934,"x2":506.429,"y2":131.648},"colorStops":[{"offset":1,"color":"rgb(242,242,242)","opacity":1},{"offset":0,"color":"rgb(102,102,102)","opacity":1}],"offsetX":0.23,"offsetY":0.23,"gradientUnits":"pixels","gradientTransform":[0.0017,0,0,0.0017,-0.966,-0.368]},"stroke":"rgb(153,153,153)","strokeWidth":0,"strokeLineCap":"round","scaleX":59.8,"scaleY":59.8,"radius":0.23}]}]}';
```

Comment:
This PR is breaking
So loading json from prev versions will not do. However exporting to svg from one version and importing to another works.
This is how I fixed this test